web/templates/settings/generate_integration_url_modal.hbs

```diff
@@ -21,6 +21,15 @@
         {{t "Send all notifications to a single topic"}}
     </label>
 </div>
+<div class="input-group hide" id="skip-private-repositories-container">
+    <label class="checkbox">
+        <input type="checkbox" id="skip-private-repositories" class="integration-url-parameter"/>
+        <span class="rendered-checkbox"></span>
+    </label>
+    <label class="inline" for="skip-private-repositories">
+        {{t "Exclude notifications from private repositories"}}
+    </label>
+</div>
```

Comment:
`id`s are supposed to be globally unique across the entire page, which is why all the other `id`s in this file are scoped with `integration-url-`. This should be scoped as well.